docs/Getting Started.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started
+
+## Clone Serai

```

Comment:
Clone Serai should be after dependencies so we can delinerate environment steps from Serai steps.